scss/functions/_get-color.scss

```diff
@@ -2,9 +2,22 @@
 @use 'sass:map';
 
 @function get-color($type, $level) {
-  // add the logic here
+  @if map.has-key($colors, $type) {
+    @if map.get($colors, $type) and map.has-key(map.get($colors, $type) , $level ) {
```

Comment:
`@if map.get($colors, $type) and ` pada line 6 bisa dihapus aja ya ` map.has-key(map.get($colors, $type) , $level ) {` dan sisakan bagian ini saja pada line 6